nextflow-base-images/approved_img_layers.json

```diff
@@ -0,0 +1,3 @@
+{
+    "_comment": "The `get_layer_info_for_nf_imgs.yml` GitHub workflow reads the `nextflow-base-images/allowed_base_images.txt` file and populates this `approved_img_layers.json` file with layer information for each base image. You are reading a placeholder file. The current list of approved layers can be found at `https://github.com/uc-cdis/containers/blob/store/branch_to_track_nf_img_layers/nextflow-base-images/approved_img_layers.json`. The workflow currently assumes that only ECR images are in the list of approved base images."
```

Comment:
it would be easier to read if it was just text instead of json and paragraphs as suggested in my previous comment.
It's ok if it's not valid JSON. it's just a placeholder

![Screenshot 2024-04-18 at 4 13 19 PM](https://github.com/uc-cdis/containers/assets/4224001/21fc97dd-292f-42e9-94b8-573a5c36e1a9)

Reply:
Okay! I can change it.